Source/Urho3D/Graphics/StaticModel.cpp

```diff
@@ -317,6 +317,13 @@ void StaticModel::ApplyMaterialList(const ea::string& fileName)
         return;
 
     const StringVector lines = file->ReadLines();
+    ApplyMaterialList(lines);
+}
+
+void StaticModel::ApplyMaterialList(const StringVector& lines)
```

Comment:
Why not to call 
    /// Set materials attribute.
    void SetMaterialsAttr(const ResourceRefList& value);
It takes ResourceRefList which is list of strings + resource type.

Reply:
I could do that, but I don't think a ResourceRefList will work with a StringVector without doing some converting. StringVector is what material lists are represented as, ModelView::ExportMaterialList is a example.

Reply:
I would say that ExportMaterialList  should return ResourceRefList then. @eugeneko @rokups thoughts, suggestions?

Reply:
I am inclined to agree. For public API `ResourceRefList` would make more sense.. `ApplyMaterialList(const StringVector& lines)` looks more like a helper function whose place is in protected/private section.

Reply:
ExportMaterialList needs to return ResourceRefList then. I agree with such a change but ExportMaterialList needs to be changed too then.